wormhole-connect/src/utils/ntt.ts

```diff
@@ -122,3 +123,31 @@ export const getNttTokenByGroupKey = (
   if (!manager) return;
   return config.tokens[manager.tokenKey];
 };
+
+// Get the NTT config for the SDK routes
+export function getNttConfig(): NttRoute.Config {
+  const result: NttRoute.Config = { tokens: {} };
+
+  Object.entries(config.nttGroups).forEach(([name, group]) => {
+    const tokens: NttRoute.TokenConfig[] = group.nttManagers
+      .map((manager) => {
+        const tokenAddress = config.tokens[manager.tokenKey]?.tokenId?.address;
+        return tokenAddress
+          ? {
+              chain: config.sdkConverter.toChainV2(manager.chainName),
+              token: tokenAddress,
+              manager: manager.address,
+              transceiver: manager.transceivers,
+              quoter: manager.solanaQuoter,
+            }
+          : null;
+      })
+      .filter((token) => token !== null);
+
+    if (tokens.length > 0) {
+      result.tokens[name] = tokens;
+    }
+  });
+
+  return result;
+}
```

Comment:
what do you think about converging on SDKv2's NTT config types in Connect's config? since this is going to be a major version change to Connect anyway, I think this is our opportunity to make some breaking changes to the `config` interface. we could just adopt the `NttRoute.Config` type in the `WormholeConnectConfig` interface.

it would be nice to be consistent across Connect & SDK for NTT for those config types and it would remove the need to add this code at all.

Reply:
I think that's a good idea, I can look into changing it

Reply:
Will do this in a follow up PR